### PR TITLE
(gce) Add support for custom machine types 

### PR DESCRIPTION
**Pull request by danielpeach**
(gce) Add support for custom machine types 
@duftler PTAL

screen shot 2016-06-20 at 1 51 35 pm

### CODE REVIEW

File: app/scripts/modules/google/serverGroup/configure/wizard/customInstance/customInstanceBuilder.gce.service.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.serverGroup.customInstanceBuilder.gce.service', [
+  require('../../../../../core/utils/lodash.js')
+])
+  .factory('gceCustomInstanceBuilderService', function(_) {
+    /*
+    * Zones that support Haswell and Ivy Bridge processors can support custom machine types up to 32 vCPUs.
```
Comment (duftler) [inline app/scripts/modules/google/serverGroup/configure/wizard/customInstance/customInstanceBuilder.gce.service.js]: Please add a link to this comment to keep it in sync with the related list in clouddriver (should link to the class file). Please also add a link to the clouddriver list pointing back here.

**Comment by duftler** [comment]
I verified that this works as expected. I also confirmed that the new Build Custom family does not show up for aws.
LGTM.

**Comment by duftler** [comment]
@anotherchrisberry Any concerns about us merging this one (it touches a couple of `core` files)?

**Comment by duftler** [comment]
Please rebase and squash commits.

**Comment by anotherchrisberry** [comment]
LGTM - nice work

**Pull request merged**